include/roboteam_ai/TreeHelper/RobotOutOfFieldHelper.h

```diff
@@ -0,0 +1,18 @@
+//
+// Created by jessevw on 21.11.19.
+//
+
+#ifndef RTT_ROBOTOUTOFFIELDHELPER_H
+#define RTT_ROBOTOUTOFFIELDHELPER_H
+#include <include/roboteam_ai/bt/Node.hpp>
+
+namespace bt {
+    class RobotOutOfFieldHelper {
+    public:
+        RobotOutOfFieldHelper();
+        std::shared_ptr<Node> createRobotOutOfFieldHelper();
```

Comment:
- why not createRobotOutOfFieldNode? (as you are returning a node) 
Or maybe an even better question: 
- why not RAII? -> Make the helper extend on Node?